virtual-env.ipynb

```diff
@@ -26,78 +31,116 @@
     "\n",
     "A Python virtual environment is a named, isolated, working copy of Python that maintains its own files, directories, and paths so that you can work with specific versions of libraries or Python itself without affecting other Python projects.\n",
     "Virtual environments make it easy to cleanly separate projects and avoid problems with different dependencies and version requirements across components.\n",
-    "The conda command-line interface (CLI) is the preferred interface for managing installations and virtual environments with the Anaconda Python distribution.\n",
-    "The Jupyter Notebook service of the Iguazio Data Science Platform (\"the platform\") comes pre-deployed with conda.\n",
-    "This tutorial explains how to use conda to create a Python virtual environment that will be available as a custom kernel in Jupyter Notebook.\n",
-    "For general information about using conda to create virtual environments, see the [conda documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)."
+    "The Conda command-line interface (CLI) is the preferred interface for managing installations and virtual environments with the Anaconda Python distribution.\n",
+    "The Jupyter service of the Iguazio Data Science Platform (\"the platform\") comes pre-deployed with Conda.\n",
+    "This tutorial explains how to use Conda to create a Python virtual environment that will be available as a custom kernel in JupyterLab.\n",
+    "For general information about using Conda to create virtual environments, see the [Conda documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html)."
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"preconfigured-conda-environments\"></a>\n",
+    "## Preconfigured Conda environments\n",
+    "\n",
+    "Iguazio provides several preconfigured Conda environments that are included in the Jupyter pod:\n",
+    "- **base**: This is the default base environment that includes the Conda binaries. It should not be used for development purposes.\n",
```

Comment:
Suggest replacing few instances of (unless there are circumstances in which you can, which should be specified):
It should not be used...
with:
Do not use it...